compiler/rustc_error_codes/src/error_codes/E0093.md

```diff
@@ -6,9 +6,9 @@ Erroneous code example:
 #![feature(intrinsics)]
 #![allow(internal_features)]
 
-extern "rust-intrinsic" {
-    fn foo(); // error: unrecognized intrinsic function: `foo`
-}
+#[rustc_intrinsic]
+unsafe fn foo(); // error: unrecognized intrinsic function: `foo`
+
```

Comment:
Is there a particular reason why you added a new empty line here (and in all the other tests)?